Gems/GradientSignal/Code/Source/Components/ImageGradientComponent.cpp

```diff
@@ -102,6 +102,123 @@ namespace GradientSignal
 
     AZ_CLASS_ALLOCATOR_IMPL(JsonImageGradientConfigSerializer, AZ::SystemAllocator, 0);
 
+    AZStd::vector<AZ::Edit::EnumConstant<ChannelToUse>> SupportedChannelOptions()
+    {
+        AZStd::vector<AZ::Edit::EnumConstant<ChannelToUse>> options;
+
+        options.push_back(AZ::Edit::EnumConstant<ChannelToUse>(ChannelToUse::R, "Red"));
+        options.push_back(AZ::Edit::EnumConstant<ChannelToUse>(ChannelToUse::G, "Green"));
+        options.push_back(AZ::Edit::EnumConstant<ChannelToUse>(ChannelToUse::B, "Blue"));
+        options.push_back(AZ::Edit::EnumConstant<ChannelToUse>(ChannelToUse::A, "Alpha"));
+        options.push_back(AZ::Edit::EnumConstant<ChannelToUse>(ChannelToUse::Terrarium, "Terrarium"));
+
+        return options;
+    }
+
+    AZStd::vector<AZ::Edit::EnumConstant<CustomScaleType>> SupportedScaleOptions()
+    {
+        AZStd::vector<AZ::Edit::EnumConstant<CustomScaleType>> options;
+
+        options.push_back(AZ::Edit::EnumConstant<CustomScaleType>(CustomScaleType::None, "None"));
+        options.push_back(AZ::Edit::EnumConstant<CustomScaleType>(CustomScaleType::Manual, "Manual"));
+
+        return options;
+    }
+
+    bool DoesFormatSupportTerrarium(AZ::RHI::Format format)
+    {
+        // The terrarium type is only supported by 8-bit formats that have
+        // at least RGB
+        switch (format)
+        {
+        case AZ::RHI::Format::R8G8B8A8_UNORM:
+        case AZ::RHI::Format::R8G8B8A8_UNORM_SRGB:
+            return true;
+        }
+
+        return false;
+    }
+
+    AZStd::pair<float, float> GetPixelFormatRange(AZ::RHI::Format format)
+    {
+        switch (format)
+        {
+        case AZ::RHI::Format::R8_UNORM:
+        case AZ::RHI::Format::A8_UNORM:
+        case AZ::RHI::Format::R8G8_UNORM:
+        case AZ::RHI::Format::R8G8B8A8_UNORM:
+        case AZ::RHI::Format::A8B8G8R8_UNORM:
+        case AZ::RHI::Format::R8_UNORM_SRGB:
+        case AZ::RHI::Format::R8G8_UNORM_SRGB:
+        case AZ::RHI::Format::R8G8B8A8_UNORM_SRGB:
+        case AZ::RHI::Format::A8B8G8R8_UNORM_SRGB:
+        case AZ::RHI::Format::R8_UINT:
+        case AZ::RHI::Format::R8G8_UINT:
+        case AZ::RHI::Format::R8G8B8A8_UINT:
+            return AZStd::make_pair(aznumeric_cast<float>(AZStd::numeric_limits<AZ::u8>::lowest()),
+                aznumeric_cast<float>(AZStd::numeric_limits<AZ::u8>::max()));
```

Comment:
I _think_ this is ok in this context, but just want to point out that srgb / unorm are technically in the 0 to 1 range, and snorm is in the -1 to 1 range. Just want to make sure it's intentional that this is just treating everything as a regular integer.